### PR TITLE
Refactored cli ex, workers, controllers, nodes, and ex commands 

### PR DESCRIPTION
**Pull request by jeffmontagna**
ex, workers, controllers, nodes, and ex now used the new config and teraslice-util classes.
- Consolidated the console display to use a single library `cli-table3` to display tables.
- Teraslice api response parsing is moved into the `teraslice-util` class instead of each command having two parsing functions. 
- Added `ex/errors` functionality, this was only stubbed out before.
- The output of `controllers/stats` is now displayed vertically table instead of h

### CODE REVIEW

File: packages/teraslice-cli/lib/teraslice-util.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            }
+        } catch (err) {
+            if (err.code === 405 && err.error === 405) {
+                clusteringType = 'native';
+            }
+        }
+        return clusteringType;
+    }
+
+    parseResponse(response, header) {
```
Comment (godber) [inline packages/teraslice-cli/lib/teraslice-util.js]: Can you provide a comment that describes the purpose of this function that distinguishes it from `parseStateResponse`?  Indicate what the input `response` comes from and what the returned value looks like.

File: packages/teraslice-cli/test/lib/teraslice-util-spec.js [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+    });
+
+    afterEach(() => {
+        cliArgs = {};
+        testTerasliceUtil = {};
+    });
+
+    test('should return a config object', () => {
+        expect(testTerasliceUtil).toBeDefined();
+    });
```
Comment (godber) [inline packages/teraslice-cli/test/lib/teraslice-util-spec.js]: Leave empty line between tests.

File: packages/teraslice-cli/test/cmds/ex/errors-spec.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+                errors.desc,
+                errors.builder,
+                () => true
+            );
+            const yargsResult = yargsCmd.parse(
+                'errors ts-test1 99999999-9999-9999-9999-999999999999', {}
+            );
+            expect(yargsResult.clusterAlias).toEqual('ts-test1');
+            expect(yargsResult.id).toEqual('99999999-9999-9999-9999-999999999999');
+        });
```
Comment (godber) [inline packages/teraslice-cli/test/cmds/ex/errors-spec.js]: leave empty line between tests/its

File: packages/teraslice-cli/lib/teraslice-util.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+                } else {
+                    row.push(response[key][item]);
+                }
+            });
+            rows.push(row);
+        });
+        return rows;
+    }
+
+    parseStateResponse(response, header, id) {
```
Comment (godber) [inline packages/teraslice-cli/lib/teraslice-util.js]: The fact that these don't use `this` might be an indication that my suggestion to put these functions here is not the right choice.

Comment (jeffmontagna) [reply packages/teraslice-cli/lib/teraslice-util.js]: Moved from `teraslice-utils` parseResponse into `display`.  Removed parseStateResponse and integrated functionality into parseResponse.

File: packages/teraslice-cli/test/cmds/ex/errors-spec.js (hunk 3 above)
Comment (jeffmontagna) [reply packages/teraslice-cli/test/cmds/ex/errors-spec.js]: removed empty lines

File: packages/teraslice-cli/test/lib/teraslice-util-spec.js (hunk 2 above)
Comment (jeffmontagna) [reply packages/teraslice-cli/test/lib/teraslice-util-spec.js]: added empty lines

File: packages/teraslice-cli/lib/teraslice-util.js (hunk 1 above)
Comment (jeffmontagna) [reply packages/teraslice-cli/lib/teraslice-util.js]: I added comments that describe the functionally of parseResponse and specified all the params and returns.

**Pull request merged**